src/App.js

```diff
@@ -74,8 +81,16 @@ function App() {
               })}
             </div>
             <div className="btn">
-              <button onClick={handlePrevPage}>前へ</button>
-              <button onClick={handleNextPage}>次へ</button>
+              {prevURL === null ? (
+                <button className="isTop">前へ</button>
+              ) : (
+                <button onClick={handlePrevPage} className="isNotTop">
```

Comment:
こちらは `handlePrevPage` のところですでに `if (!prevURL) return;` で弾く仕様になっていますので以下のような書き方で大丈夫です
```suggestion
<button  className={prevURL == null ? "isTop" : "isNotTop"} onClick={handlePrevPage}>
```